particles/PipeApps2/source/CaptionThis.js

```diff
@@ -16,14 +16,30 @@ defineParticle(({DomParticle, html, log}) => {
     get template() {
       return `<span>{{json}}</span>`;
     }
-    render({query}) {
```

Comment:
Fwiw, using `render` instead of `update` is a trick to avoid SpecEx running expensive operations (like ML inferencing) when using Planner.

We aren't using Planner atm, so this change is fine, just want you to have the info.